### PR TITLE
Added support for Common Log Format

### LINKED ISSUE #553: Add support for Apache Common Log Format
## Detailed Description

https://httpd.apache.org/docs/2.4/logs.html#accesslog

## Context

> This standard format can be produced by many different web servers and read by many log analysis programs.

## Possible Implementation

```
127.0.0.1 - frank [10/Oct/2000:13:55:36 -0700] "GET /apache_pb.gif HTTP/1.0" 200 2326
```

### PR DESCRIPTION
**Pull request by whiskeysierra**
## Motivation and Context

Fixes #553

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: logbook-core/src/main/java/org/zalando/logbook/CommonsLogFormatSink.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+import static java.time.temporal.ChronoField.HOUR_OF_DAY;
+import static java.time.temporal.ChronoField.MINUTE_OF_HOUR;
+import static java.time.temporal.ChronoField.MONTH_OF_YEAR;
+import static java.time.temporal.ChronoField.SECOND_OF_MINUTE;
+import static java.time.temporal.ChronoField.YEAR;
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+/**
+ * @see <a href="https://en.wikipedia.org/wiki/Common_Log_Format">Wikipedia: Common Log Format</a>
+ * @see <a href="https://httpd.apache.org/docs/trunk/logs.html#common>Apache HTTP Server: Common Log Format</a>
```
Comment (lukasniemeier-zalando) [inline logbook-core/src/main/java/org/zalando/logbook/CommonsLogFormatSink.java]: missing closing `"` on the href

**Comment by lukasniemeier-zalando** [comment]
:+1:

File: logbook-core/src/main/java/org/zalando/logbook/CommonsLogFormatSink.java (hunk 1 above)
Comment (whiskeysierra) [reply logbook-core/src/main/java/org/zalando/logbook/CommonsLogFormatSink.java]: :eagle: :eye:

Comment (whiskeysierra) [reply logbook-core/src/main/java/org/zalando/logbook/CommonsLogFormatSink.java]: Fixed

Comment (lukasniemeier-zalando) [reply logbook-core/src/main/java/org/zalando/logbook/CommonsLogFormatSink.java]: It killed the syntax highlighting of the class, therefore easy to spot for me :wink:

**Comment by lukasniemeier-zalando** [comment]
:+1:

**Pull request merged**